frontend/src/components/common/sidebar/sidebar-container.tsx

```diff
@@ -0,0 +1,31 @@
+import { FC } from 'common/types/types';
+import { Sidebar } from './sidebar';
+import { subscribesList } from './subscription-list.mock';
+import { configRoutePages } from './route-pages.config';
+import { useLocation } from 'react-router-dom';
+
+function returnIdActiveRoute(currentRoute: string): number {
+  let idRoute = 0;
+
+  configRoutePages.forEach((route) => {
+    if (route.linkTo === currentRoute) {
+      idRoute = route.id;
+    }
+  });
+
+  return idRoute;
+}
```

Comment:
Use `find` instead

Reply:
Looks like you solution will not work with bigger route name. Like `/history/route-name/route-name`